lib/models/movie.dart

```diff
@@ -0,0 +1,56 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String originalLanguage;
+  final String releaseDate;
+  final num voteAverage;
+  final int voteCount;
+  final List<int> genres;
+  final String overview;
+  final String backdrop;
+  final bool rated;
+
+  Movie({
+    required this.title,
+    required this.originalTitle,
+    required this.originalLanguage,
+    required this.releaseDate,
+    required this.voteAverage,
+    required this.voteCount,
+    required this.genres,
+    required this.overview,
+    required this.backdrop,
+    required this.rated,
+  });
+
+  factory Movie.mockMovie() {
+    return Movie(
+      title: 'The Super Mario Bros. Movie',
+      originalTitle: 'The Super Mario Bros. Movie',
+      originalLanguage: 'English',
+      releaseDate: '2023-04-05',
+      voteAverage: 7.8,
+      voteCount: 2586,
+      genres: [1],
+      overview:
+          'While working underground to fix a water main, Brooklyn plumbers and brothers Mario and Luigi are transported down a mysterious pipe and wander into a magical new world. But when the brothers are separated, Mario embarks on an epic quest to find Luigi.',
+      backdrop: 'assets/images/mariobros2.jpg',
+      rated: false,
+    );
+  }
+
+  factory Movie.fromJson(json) {
+    return Movie(
+      title: json['title'],
+      originalTitle: json['original_title'],
+      originalLanguage: json['original_language'],
+      releaseDate: json['release_date'],
+      voteAverage: json['vote_average'],
+      voteCount: json['vote_count'],
+      genres: json['genre_ids'].cast<int>(),
+      overview: json['overview'],
+      backdrop: 'https://image.tmdb.org/t/p/w500${json['backdrop_path']}',
```

Comment:
Here you can only get que relative path for the backgrop and create a new method to get the full url like
`String get fullBackdrop => 'https://image.tmdb.org/t/p/w500$backdrop`
Also add the base url as a const

Reply:
Did you forgot to map the poster path ?

Reply:
Done. 
I had to make the new method static because otherwise I get an "Instance members can´t be accessed from a factory constructor" error. Is that right? Or is that I had to get the full backdrop string outside the constructor? 
On the other hand, I didn´t map the poster path because I don´t use it (as well as I don´t use other json fields). Am I supposed to get them all?

Reply:
Map the poster is an optional for this exercise, but remember to add it when you call the real service from TheMovieDB @agussbergamo